<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SistemaLogin</class>
 <widget class="QWidget" name="SistemaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>408</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>408</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>587</width>
    <height>408</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Integração Ponto REP :: Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
	background-image: url(:/img/fundoBG.png);
}

QGroupBox {
	background-image: url(:/img/fundoBG.png);
}


/* Nice Windows-XP-style password character. */
QLineEdit[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}

/* We provide a min-width and min-height for push buttons
   so that they look elegant regardless of the width of the text. */
QPushButton {
    background-color: #FFFFFF;
    border-width: 0px;
    border-color: #FF3300;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: #66CCFF;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #66FFFF;
}

QLabel, QAbstractButton {
    font: bold;
}

/* Mark mandatory fields with a brownish color. */
.mandatory {
    color: #66CCFF;
}

/* Custumizando combobox*/
QComboBox {
}
/* Bold text on status bar looks awful. */
QStatusBar QLabel {
   font: normal;
}

QStatusBar::item {
    border-width: 0;
    border-color: #FF3300;
    border-style: solid;
    border-radius: 1;
}

QComboBox, QLineEdit, QSpinBox, QTextEdit, QListView {
    background-color: #FFFFFF;
    selection-color: #003399; 
    selection-background-color: #66FFFF;
}

QListView {
    show-decoration-selected: 1;
}

QListView::item:hover {
    background-color: #66FFFF;
}

/* We reserve 1 pixel space in padding. When we get the focus,
   we kill the padding and enlarge the border. This makes the items
   glow. */
QLineEdit, QFrame, QComboBox, SpinBox, CheckBox {
    border-width: 0px;
    padding: 1px;
    border-style: solid;
    border-color: #FF3300;
    border-radius: 2px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus, QFrame:focus {
    border-width: 0.5px;
    padding: 0px;
}

/* A QLabel is a QFrame ... */
QLabel {
    border: none;
    padding: 0;
    background: none;
}

/* A QToolTip is a QLabel ... */
QToolTip {
    border: 0px solid #FF3300;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

/* Nice to have the background color change when hovered. */
QRadioButton:hover, QCheckBox:hover {
    background-color: #99CCFF;
}

/* Force the dialog's buttons to follow the Windows guidelines. */
QDialogButtonBox {
    button-layout: 0;
}


</string>
  </property>
  <widget class="QGroupBox" name="grupoGeral">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>591</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="labelUsuario">
      <property name="text">
       <string>CPF :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="campoSenha">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="botaoSair">
      <property name="text">
       <string>Sair</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelSenha">
      <property name="text">
       <string>Senha :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="campoUsuario">
      <property name="inputMethodHints">
       <set>Qt::ImhFormattedNumbersOnly</set>
      </property>
      <property name="inputMask">
       <string>000.000.000-00</string>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="botaoCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="botaoEntrar">
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="mensagemUsuario">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>campoUsuario</tabstop>
  <tabstop>campoSenha</tabstop>
  <tabstop>botaoEntrar</tabstop>
  <tabstop>botaoCancelar</tabstop>
  <tabstop>botaoSair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
